src/extensions/core/nodeBadge.ts

```diff
@@ -1,150 +0,0 @@
-  const nodeDef = node.constructor.nodeData
-  // Frontend-only nodes don't have nodeDef
-  if (!nodeDef) {
-    return null
-  }
-  const nodeDefStore = useNodeDefStore()
-  return nodeDefStore.nodeDefsByName[nodeDef.name]?.nodeSource ?? null
-}
-
-function isCoreNode(node: LGraphNode) {
-  return getNodeSource(node)?.type === NodeSourceType.Core
-}
-
-function badgeTextVisible(node: LGraphNode, badgeMode: NodeBadgeMode): boolean {
-  return (
-    badgeMode === NodeBadgeMode.None ||
-    (isCoreNode(node) && badgeMode === NodeBadgeMode.HideBuiltIn)
-  )
-}
-
-function getNodeIdBadgeText(node: LGraphNode, nodeIdBadgeMode: NodeBadgeMode) {
-  return badgeTextVisible(node, nodeIdBadgeMode) ? '' : `#${node.id}`
-}
-
-function getNodeSourceBadgeText(
-  node: LGraphNode,
-  nodeSourceBadgeMode: NodeBadgeMode
-) {
-  const nodeSource = getNodeSource(node)
-  return badgeTextVisible(node, nodeSourceBadgeMode)
-    ? ''
-    : nodeSource?.badgeText ?? ''
-}
-
-function getNodeLifeCycleBadgeText(
-  node: LGraphNode,
-  nodeLifeCycleBadgeMode: NodeBadgeMode
-) {
-  let text = ''
-  const nodeDef = node.constructor.nodeData
-
-  // Frontend-only nodes don't have nodeDef
-  if (!nodeDef) {
-    return ''
-  }
-
-  if (nodeDef.deprecated) {
-    text = '[DEPR]'
-  }
-
-  if (nodeDef.experimental) {
-    text = '[BETA]'
-  }
-
-  return badgeTextVisible(node, nodeLifeCycleBadgeMode) ? '' : text
-}
-
-class NodeBadgeExtension implements ComfyExtension {
-  name = 'Comfy.NodeBadge'
-
-  constructor(
-    public nodeIdBadgeMode: ComputedRef<NodeBadgeMode> | null = null,
-    public nodeSourceBadgeMode: ComputedRef<NodeBadgeMode> | null = null,
-    public nodeLifeCycleBadgeMode: ComputedRef<NodeBadgeMode> | null = null,
-    public colorPalette: ComputedRef<Palette> | null = null
-  ) {}
-
-  init(app: ComfyApp) {
-    const settingStore = useSettingStore()
-    this.nodeSourceBadgeMode = computed(
-      () =>
-        settingStore.get('Comfy.NodeBadge.NodeSourceBadgeMode') as NodeBadgeMode
-    )
-    this.nodeIdBadgeMode = computed(
-      () => settingStore.get('Comfy.NodeBadge.NodeIdBadgeMode') as NodeBadgeMode
-    )
-    this.nodeLifeCycleBadgeMode = computed(
-      () =>
-        settingStore.get(
-          'Comfy.NodeBadge.NodeLifeCycleBadgeMode'
-        ) as NodeBadgeMode
-    )
-    this.colorPalette = computed(() =>
-      getColorPalette(settingStore.get('Comfy.ColorPalette'))
-    )
-
-    watch(this.nodeSourceBadgeMode, () => {
-      app.graph.setDirtyCanvas(true, true)
-    })
-
-    watch(this.nodeIdBadgeMode, () => {
-      app.graph.setDirtyCanvas(true, true)
-    })
-    watch(this.nodeLifeCycleBadgeMode, () => {
-      app.graph.setDirtyCanvas(true, true)
-    })
-  }
-
-  nodeCreated(node: LGraphNode, app: ComfyApp) {
-    node.badgePosition = BadgePosition.TopRight
-    // @ts-expect-error Disable ComfyUI-Manager's badge drawing by setting badge_enabled to true. Remove this when ComfyUI-Manager's badge drawing is removed.
```

Comment:
ComfyUI-Manager supports the detection of new frontend, so we can drop this line.

https://github.com/ltdrdata/ComfyUI-Manager/commit/3701246fb16b5ecbb6f46824ce15ccf1a8664f4d